### PR TITLE
chore:(plasma-infra) bump engines npm package.json property 

### PR DESCRIPTION
**Pull request by Yakutoc**
Плановый переход на более новую версию **npm**.

Меняем настройки, что бы CI workflow работали корректно.  

--------

:baby_chick: Download canary assets:

[color_sberHealth_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.522.5067667684.swift)  
[color_sberHealth_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.522.5067667684.kt)  
[color_sberHealth_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.522.5067667684.ts)  
[color_sbermarket_business_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.522.5067667684.swift)  
[color_sbermarket_business_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.522.5067667684.kt)  
[color_sbermarket_business_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.522.5067667684.ts)  
[color_sbermarket_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.522.5067667684.swift)  
[color_sbermarket_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.522.5067667684.kt)  
[color_sbermarket_metro_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.522.5067667684.swift)  
[color_sbermarket_metro_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.522.5067667684.kt)  
[color_sbermarket_metro_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.522.5067667684.ts)  
[color_sbermarket_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.522.5067667684.ts)  
[color_sbermarket_selgros_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.522.5067667684.swift)  
[color_sbermarket_selgros_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.522.5067667684.kt)  
[color_sbermarket_selgros_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.522.5067667684.ts)  
[color_sbermarket_wlbusiness_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.522.5067667684.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.522.5067667684.kt)  
[color_sbermarket_wlbusiness_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.522.5067667684.ts)  
[color_sberonline_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.522.5067667684.swift)  
[color_sberonline_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.522.5067667684.kt)  
[color_sberonline_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.522.5067667684.ts)  
[color_sberprime_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.522.5067667684.swift)  
[color_sberprime_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.522.5067667684.kt)  
[color_sberprime_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.522.5067667684.ts)  
[shadow_sbermarket_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.522.5067667684.ts)  
[typo_mage_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.522.5067667684.swift)  
[typo_mage_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.522.5067667684.kt)  
[typo_mage_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.522.5067667684.ts)  
[typo_plasma_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.522.5067667684.swift)  
[typo_plasma_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.522.5067667684.kt)  
[typo_plasma_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.522.5067667684.ts)  
[typo_ruler_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.522.5067667684.swift)  
[typo_ruler_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.522.5067667684.kt)  
[typo_ruler_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.522.5067667684.ts)  
[typo_sage_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.522.5067667684.swift)  
[typo_sage_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.522.5067667684.kt)  
[typo_sage_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.522.5067667684.ts)  
[typo_sbermarket_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.522.5067667684.swift)  
[typo_sbermarket_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.522.5067667684.kt)  
[typo_sbermarket_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.522.5067667684.ts)  
[typo_soulmate_ios-swift--canary.522.5067667684.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.522.5067667684.swift)  
[typo_soulmate_kotlin--canary.522.5067667684.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.522.5067667684.kt)  
[typo_soulmate_react-native--canary.522.5067667684.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.522.5067667684.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.215.1-canary.522.5067667684.0
  npm install @salutejs/plasma-temple@1.164.1-canary.522.5067667684.0
  npm install @salutejs/plasma-ui@1.196.1-canary.522.5067667684.0
  npm install @salutejs/plasma-web@1.215.1-canary.522.5067667684.0
  # or 
  yarn add @salutejs/plasma-b2c@1.215.1-canary.522.5067667684.0
  yarn add @salutejs/plasma-temple@1.164.1-canary.522.5067667684.0
  yarn add @salutejs/plasma-ui@1.196.1-canary.522.5067667684.0
  yarn add @salutejs/plasma-web@1.215.1-canary.522.5067667684.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-522/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-522/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-522/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-522/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-522/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-522/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-522/temple-storybook/

**Review by kayman233 — APPROVED** [review]

**Review by neretin-trike — APPROVED** [review]

File: .npmrc [hunk 1]
```diff
@@ -5,7 +5,4 @@ save-exact=true
 # disabled auto-installing peer dependencies
 legacy-peer-deps=true
 
-# for specify Node, NPM Version to use
-engine-strict=true
```
Comment (Yeti-or) [inline .npmrc:9]: я не понял почему отпала

Comment (Yakutoc) [reply .npmrc:9]: Было добавлено когда у нас использовалась **конкретная** версия npm.

В этом ПР я убрал поле `package.engines`, а без него опция `engine-strict` бессмысленна. 

ИМХО так как теперь у нас будет 8.19.2 версия npm ограничения не имеют смысла.

Comment (Yakutoc) [reply .npmrc]: После обсуждения было решено оставить эти настройки, но изменить версию npm&

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-522/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-522/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-522/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-522/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-522/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-522/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-522/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-522/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-522/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-522/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-522/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-522/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-522/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-522/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-522/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-522/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-522/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-522/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-522/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-522/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-522/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

**Pull request merged**